### ISSUE TITLE
Plans for elm 0.19?

### ISSUE THREAD

**Issue by ryan-senn** [eeue56/elm-all-dict#6]
Are there any plans to migrate this to elm 0.19? Also does the removal of Basics.toString affect this at all?

**Comment by Erudition** [comment]
> Are there any plans to migrate this to elm 0.19? Also does the removal of Basics.toString affect this at all?

Yep, looks like that very change makes this library's core concept impossible in production.

**Comment by skyqrose** [comment]
I use [pzp1997/assoc-list](https://package.elm-lang.org/packages/pzp1997/assoc-list/latest) instead. It's the same idea, a replacement for `Dict` that allows non-`comparable` keys, though it has different constraints and features than this package.

**Comment by Erudition** [comment]
Thanks for that @skyqrose, that looks like the way to go.
